### PR TITLE
Update README - "About this project"

### PR DESCRIPTION
**Pull request by yangyansong-adbe**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# AEPCore
+# Adobe Experience Platform SDK
```
Comment (swarna04) [inline README.md:1]: Should this be Adobe Experience Platform Core SDK?

File: README.md [hunk 2]
```diff
@@ -15,8 +15,12 @@ By using the Beta, you hereby acknowledge that the Beta is provided "as is" with
 
 ## About this project
 
-The Mobile Core represents the core Adobe Experience Platform SDK that is required for every app implementation. The core contains a common set of functionality and frameworks, such as  Experience Cloud Identity services, data event hub, Rules Engine, reusable networking, disk access routines, and so on, which are required by all Adobe and third-party extensions.
+`AEPCore`, `AEPService`, `AEPIdentity` represent the core Adobe Experience Platform SDK that are required for every app implementation. The core extensions contain a common set of functionality and frameworks, such as  Experience Cloud Identity services, data event hub, reusable networking, disk access routines, and so on, which are required by all Adobe and third-party extensions.
```
Comment (swarna04) [inline README.md:18]: ...represent the core Adobe Experience Platform SDK **extensions** that...

File: README.md [hunk 3]
```diff
@@ -15,8 +15,12 @@ By using the Beta, you hereby acknowledge that the Beta is provided "as is" with
...
 
+`AEPSignal` represents the Adobe Experience Platform SDK `Signal` extension that allows marketers to send a "signal" to their apps through the Experience Platform SDKs.
```
Comment (swarna04) [inline README.md:20]: Should we rephrase this to provide some more info?
```
`AEPSignal` represents the Adobe Experience Platform SDK's `Signal` extension that allows marketers to send a "signal" to their apps to send data to external destinations or to open URLs. 
```

File: README.md [hunk 4]
```diff
@@ -15,8 +15,12 @@ By using the Beta, you hereby acknowledge that the Beta is provided "as is" with
 
 ## About this project
 
-The Mobile Core represents the core Adobe Experience Platform SDK that is required for every app implementation. The core contains a common set of functionality and frameworks, such as  Experience Cloud Identity services, data event hub, Rules Engine, reusable networking, disk access routines, and so on, which are required by all Adobe and third-party extensions.
+`AEPCore`, `AEPService`, `AEPIdentity` represent the core Adobe Experience Platform SDK extensions that are required for every app implementation. The core extensions contain a common set of functionality and frameworks, such as  Experience Cloud Identity services, data event hub, reusable networking, disk access routines, and so on, which are required by all Adobe and third-party extensions.
 
+`AEPSignal` represents the Adobe Experience Platform SDK `Signal` extension that allows marketers to send a "signal" to their apps through the Experience Platform SDKs.
+
+
+`AEPLifecycle` represents the Adobe Experience Platform SDK `Lifecycle` extension that provides out-of-the-box, application lifecycle information about your app user.
```
Comment (swarna04) [inline README.md:23]:
```
`AEPLifecycle` represents the Adobe Experience Platform SDK's `Lifecycle` extension that helps collect application Lifecycle metrics such as, application install or upgrade information, application launch and session information, device information, and any additional context data provided by the application developer. 
```

File: README.md [hunk 5]
```diff
@@ -15,8 +15,12 @@ By using the Beta, you hereby acknowledge that the Beta is provided "as is" with
 
 ## About this project
 
-The Mobile Core represents the core Adobe Experience Platform SDK that is required for every app implementation. The core contains a common set of functionality and frameworks, such as  Experience Cloud Identity services, data event hub, Rules Engine, reusable networking, disk access routines, and so on, which are required by all Adobe and third-party extensions.
+`AEPCore`, `AEPService`, `AEPIdentity` represent the core Adobe Experience Platform SDK extensions that are required for every app implementation. The core extensions contain a common set of functionality and frameworks, such as  Experience Cloud Identity services, data event hub, reusable networking, disk access routines, and so on, which are required by all Adobe and third-party extensions.
```
Comment (shalehaha) [inline README.md:18]: @sbenedicadb do you want to provide better wordings for this segment?

File: .github/workflows/release.yml [hunk 6]
```diff
@@ -8,6 +8,41 @@ on:
...
+        required: true
+        default: 'yes' 
+      
+      action_tag:
+        description: 'create tag ("no" to skip)'     
+        required: true
+        default: 'yes'
+        
+      release_AEPService:
+        description: 'release AEPService ("no" to skip)'
```
Comment (swarna04) [inline .github/workflows/release.yml:23]: AEPServices here too

File: .github/workflows/release.yml [hunk 7]
```diff
@@ -8,6 +8,41 @@ on:
...
+        description: 'Pod/SPM verification ("no" to skip)'     
+        required: true
+        default: 'yes' 
+      
+      action_tag:
+        description: 'create tag ("no" to skip)'     
+        required: true
+        default: 'yes'
+        
+      release_AEPService:
```
[the remark this answers is not in the document]
Comment (yangyansong-adbe) [reply .github/workflows/release.yml]: @swarna04  I'd update it in a separate PR since it's not related to documentation changes.

File: README.md (hunk 5 above)
Comment (sbenedicadb) [reply README.md:12]: The `AEPCore`, `AEPServices`, and `AEPIdentity` extensions represent the foundation of the Adobe Experience Platform SDK - every app using the SDK must include them. These modules contain a common set of functionality and services which are required by all SDK extensions.

`AEPCore` contains implementation of the Event Hub. The Event Hub is the mechanism used for delivering events between the app and the SDK. The Event Hub is also used for sharing data between extensions.

`AEPServices` provides several reusable implementations needed for platform support, including networking, disk access, and database management.

`AEPIdentity` implements the integration with Adobe Experience Platform Identity services.

File: README.md [hunk 8]
```diff
@@ -1,4 +1,4 @@
-# AEPCore
+# Adobe Experience Platform Core SDK
 
 <!--
 on [![Cocoapods](https://img.shields.io/cocoapods/v/AEPCore.svg?color=orange&label=AEPCore&logo=apple&logoColor=white)](https://cocoapods.org/pods/AEPCore)
```
Comment (sbenedicadb) [inline README.md:4]: when we add the cocoapods badges up here, let's make sure we add them for each of the extensions in this repo
